data/braxas.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "Töne, die Freude machen",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
+  "avatar": "https://xn--tnediefreudemachen-d3b.de/wp-content/uploads/2021/11/fav.jpg"
```

Please add a missing comma `,` here to validate your json! 

```suggestion
  "avatar": "https://xn--tnediefreudemachen-d3b.de/wp-content/uploads/2021/11/fav.jpg",
```